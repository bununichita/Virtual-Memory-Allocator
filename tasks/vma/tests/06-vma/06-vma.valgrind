==8882== Memcheck, a memory error detector
==8882== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==8882== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==8882== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==8882== Parent PID: 8881
==8882== 
==8882== 
==8882== HEAP SUMMARY:
==8882==     in use at exit: 0 bytes in 0 blocks
==8882==   total heap usage: 19 allocs, 19 frees, 8,571 bytes allocated
==8882== 
==8882== All heap blocks were freed -- no leaks are possible
==8882== 
==8882== For lists of detected and suppressed errors, rerun with: -s
==8882== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
