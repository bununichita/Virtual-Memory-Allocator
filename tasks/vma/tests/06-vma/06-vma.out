Total memory: A bytes
Free memory: 7 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 3

Block 1 begin
Zone: 1 - 4
Miniblock 1:	1	-	2	| RW-
Miniblock 2:	2	-	3	| RW-
Miniblock 3:	3	-	4	| RW-
Block 1 end
